docs/overview/04-proof-aggregation/04-domain-management.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Domain Management
+---
+
+A domain is identified by an identifier and identifies a class of aggregation and its key properties
+
+- Aggregation size
+- Publishing queue size
+
+Everyone can create a new domain and become the owner. The owner should hold some founds in order to create a new domain, the hold amount depends on the size of both aggregation and queue that the domain define and these found will be released when the domain is removed.
+
+## Create a new domain
+
+To create a new domain the owner should call [`registerDomain(aggregationSize, queueSize)`](../02-mainchain/05-mainchain_api.md#registerdomain) extrisic and define both aggregation size and queue size.
+
+The aggregation size should be any unsigned integer smaller than 128, but because it represents a Merkle tree size is better to use a power of two. From the aggregation size a proof submitter that would use this domain can estimate what will be the cost of check his proof presence in the aggregation receipt in the destination chain where he wants to use his proof: a bigger size means a bigger cost on check it, but also a lower cost send the proof on the destination chain due the bridge cost is amortized by more proofs in the same attestation.
```

Comment:
```suggestion
The aggregation size should be any unsigned integer smaller than 128, but because it represents a Merkle tree size is better to use a power of two. From the aggregation size a proof submitter that would use this domain can estimate what will be the cost of the membership proof in the aggregation receipt in the destination chain where he wants to use his proof: a bigger size means a bigger cost, but also a lower cost to send the proof on the destination chain since the bridging cost will be amortized by more proofs in the same attestation.
```

Reply:
Haven't we already reasoned about the costs in one of the previous sections ?